src/Http/Cookie/Cookie.php

```diff
@@ -621,7 +621,7 @@ public function withNeverExpire()
     public function withExpired()
     {
         $new = clone $this;
-        $new->expiresAt = new DateTimeImmutable('1970-01-01 00:00:01');
+        $new->expiresAt = new DateTimeImmutable('1970-01-01 00:00:01', new DateTimeZone('UTC'));
```

Comment:
This seems quite redundant. Expiring/delete a cookie just requires setting a time in the past and `1970-01-01 00:00:01` will always be in the past regardless of the timezone.

Reply:
Not exactly. if "expire" timestamp in cookie is negative, the expires is not effective in browser.
I don't know why but i have made an example in the issue.

Reply:
Ah, for time zones behind UTC using this datetime string can generate a negative timestamp. We can change it to `new DateTimeImmutable('@1')` instead of creating timezone instance.

Reply:
(maybe php setcookie() function bug)